src/variant.ts

```diff
@@ -0,0 +1,93 @@
+import { Context } from './context';
+import { FeatureInterface } from './feature';
+import * as murmurHash3 from 'murmurhash3js';
+
+enum PayloadType {
+    STRING = 'string',
+}
+
+export interface PayloadInterface {
+    type: PayloadType;
+    value: string;
+}
+
+export interface VariantInterface {
+    name: string;
+    weight: number;
+    payload: PayloadInterface;
+}
+
+export interface VariantResult {
+    name: string;
+    payload: PayloadInterface;
+    enabled: boolean;
+}
+
+export function getDefaultVariantResult(): VariantResult {
```

Comment:
I would like to simply call this `Variant`.

Reply:
done